Templates/login.html

```diff
@@ -8,12 +8,17 @@
         <div class="forms">
             <div class="container">
             <h1>Zaloguj się:</h1>
-            <form>
+            <form action="http://127.0.0.1:5000/login/" method="POST">
```

Comment:
```suggestion
            <form action="/login/" method="POST">
```
Wtedy zadziała na każdym adresie ;)